SVPulsingAnnotationView/SVPulsingAnnotationView.m

```diff
@@ -53,7 +53,7 @@ - (id)initWithAnnotation:(id<MKAnnotation>)annotation reuseIdentifier:(NSString
     return self;
 }
 
-- (void)rebuildLayers {
+- (void)recreateLayers {
```

Comment:
I understand why you did this, but I'd avoid creating another method. Other devs won't know which method to call. Create a block inside `rebuildLayers` and call it at the appropriate place.